File: week-03/day-03/07.py
# create a function that takes a list and returns a new list with all the elements doubled
ag = [3, 4, 5, 6, 7]

### value double ###
def value_double(value):
    k = []
    for i in value:
        k += [i * 2]
    return k
print(value_double(ag))

### element double ###
def element_double (input):
    return input * 2
print(element_double(ag))

File: week-09/day-02/07.js
'use strict';

var numbers = [2, 5, 11, 29];

// create a function that takes an array of numbers and returns a boolean
// it should return true if all the elements are prime, false otherwise

function list_checker(list) {
  return list.every(function(i) { return is_prime(i)});
}

function is_prime(number) {
    for(var i = 2; i < number; i++) {
        if(number % i === 0) {
            return false;
        }
    }
    return number > 1;
}

console.log(list_checker(numbers))
